Models/Crime.cs

```diff
@@ -21,133 +13,175 @@ namespace Neighbourhood_Alert
     using Newtonsoft.Json;
     using Newtonsoft.Json.Converters;
 
-    public partial class NeighbourhoodAlert
+    public partial class Crime
+
     {
+        private string instanceid;
+        private string incidentNo;
+        private DateTimeOffset dateReported;
+        private DateTimeOffset dateFrom;
+        private DateTimeOffset dateTo;
+        private Clsd? clsd;
+        private long ucr;
+        private DstUnion dst;
+        private long? beat;
+        private string offense;
+        private string location;
+        private TheftCode? theftCode;
+        private HateBias hateBias;
+        private Dayofweek? dayofweek;
+        private long? rptArea;
+        private string cpdNeighborhood;
+        private string neighborhood;
+        private Weapons weapons;
+        private DateTimeOffset? dateOfClearance;
+        private string hourFrom;
+        private string hourTo;
+        private string addressX;
+        private string longitudeX;
+        private string latitudeX;
+        private Age victimAge;
+        private Race? victimRace;
+        private Ethnicity? victimEthnicity;
+        private VictimGender? victimGender;
+        private Age suspectAge;
+        private Race? suspectRace;
+        private Ethnicity? suspectEthnicity;
+        private SuspectGender? suspectGender;
+        private long totalnumbervictims;
+        private long? totalsuspects;
+        private UcrGroup ucrGroup;
+        private string communityCouncilNeighborhood;
+        private long zip;
+        private Floor? floor;
+        private Side? side;
+        private Opening? opening;
+
         [JsonProperty("instanceid")]
-        public string Instanceid { get; set; }
```

Comment:
Encapsulation is a good idea, though {get; set; } syntax is often considered a shortcut for encapsulation.  In our class, I'm fine with either approach.